omnibus/files/private-chef-cookbooks/private-chef/templates/default/bookshelf.config.erb

```diff
@@ -12,6 +12,14 @@
          {error_logger_mf_maxbytes, <%= @log_rotation['file_maxbytes'] %>},
          {error_logger_mf_maxfiles, <%= @log_rotation['num_to_keep'] %>}
         ]},
+
+ {crypto, [
+           <% if node['private_chef']['fips_enabled'] -%>
+                {fips_mode, true}
```

Comment:
Could this safely be `{fips_mode, <%= node['private_chef']['fips_enabled'] %>}` instead of a conditional?

Reply:
indeed :)